### PR TITLE
November 2019 blog post

### PR DESCRIPTION
**Pull request by third774**
@chungthuang & @pyxld-kris

### CODE REVIEW

**Comment by lawnsea** [comment]
LGTM!

File: _posts/2019-11-18-meetup.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: DevLaunchers - world's most FUN experiment to help teenagers learn game development!
+author: kevin
+layout: post
+categories:
+- education
+- games
+- volunteering
+when: 2019-11-19T19:30:00-05:00
```
Comment (lawnsea) [inline _posts/2019-11-18-meetup.md]: Time is off by one hour. I guess this should be `-04:00`, @aaronj1335?

Comment (third774) [reply _posts/2019-11-18-meetup.md]: Dang daylight savings time.

Comment (third774) [reply _posts/2019-11-18-meetup.md]: Looks like it should be `-06:00`

https://time.is/Austin#time_zone

Comment (third774) [reply _posts/2019-11-18-meetup.md]: https://github.com/austinjavascript/austinjavascript.github.io/pull/101

**Pull request merged**